api/app/signals/apps/signals/managers.py

```diff
@@ -17,16 +17,19 @@
 
 
 class AttachmentManager(models.Manager):
-
     def get_attachments(self, signal):
-        from signals.apps.signals.models import Attachment
+        from .models import Signal
+        signal_id = signal.pk if isinstance(signal, Signal) else signal
 
-        return Attachment.objects.filter(_signal=signal).order_by('created_at')
+        return self.get_queryset().filter(_signal_id=signal_id).order_by('created_at')
```

Comment:
Actually the use `_signal_id` is faster because this will acces the field stored in the database. The `_signal__id` will do a filter based on the data in the related table.